integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -27,6 +27,7 @@ public class APISteps {
     private static final String DEFAULT_REDIRECT_URI =
             "https://di-ipv-core-stub.london.cloudapps.digital/callback";
     private static final String DEFAULT_CLIENT_ID = "ipv-core-stub";
+    private String authorizationCode;
```

Comment:
let's rename this to `currentAuthorizationCode` so that it matches the naming convention used for `currentSessionId`